bin/proving-service/README.md

```diff
@@ -1,8 +1,8 @@
 # Miden proving service
 
-A service for generating Miden proofs on-demand. The binary enables spawning workers and a proxy for Miden's remote proving service. Currently, only supports transactions proving.
+A service for generating Miden proofs on-demand. The binary enables spawning workers and a proxy for Miden's remote proving service. Currently, only supports transactions and batch proving.
```

Comment:
I would change the last sentence to "Currently supports proving individual transactions and transaction batches."